java/org/apache/jasper/EmbeddedServletOptions.java

```diff
@@ -84,6 +84,11 @@
      */
     private boolean classDebugInfo = true;
 
+    /**
+     * Do we want to Trim White Space in the in JSP Files?
+     */
+    private boolean JSPWhiteSpaceTrimming = false;
```

Comment:
`jspWhiteSpaceTrimming` looks better to me. It follows Java coding conventions

Reply:
What is the difference with https://github.com/apache/tomcat/blob/6aef6f468c843f2bcdec4b57884d1e7e65c03365/java/org/apache/jasper/EmbeddedServletOptions.java#L601 ?

Reply:
trimSpaces will only touch the whitespaces. Examples: 
 Node text =
1 )  "          " => ""
2) \n\n\n => ""
3) <title>\n\n\n\n => <title>\n\n\n\n (untouched)

jspWhiteSpaceTrimming will touch all of the above and reduce all the excess whitespace the JSP page might have.

Reply:
Resolved with latest push

Reply:
@markt-asf Have you read these comments ? 
trimSpace only touches white spaces / empty outputs.